### PR TITLE
feat(card): Added CardControl

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
GitHub Issue: #14 

## PR Type

What kind of change does this PR introduce?
- Feature

## Description

Added CardControl

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [X] Tested UWP
- [X] Tested iOS
- [X] Tested Android
- [X] Tested WASM
- [x] Contains **No** breaking changes

## Other information

Still needed to be done in another PR:
- Elevation (with ElevatedView)

### CODE REVIEW

File: src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/SamplesPage.xaml.cs [hunk 1]
```diff
@@ -52,6 +52,8 @@ private void NavView_Loaded(object sender, RoutedEventArgs e)
 			{ Content = "ToggleSwitch Overview", Icon = new SymbolIcon(Symbol.Next), Tag = "ToggleSwitchSamplePage" });
 			NavView.MenuItems.Add(new NavigationViewItem()
 			{ Content = "FAB Overview", Icon = new SymbolIcon(Symbol.Next), Tag = "FabSamplePage" });
+			NavView.MenuItems.Add(new NavigationViewItem()
+			{ Content = "Cards Overview", Icon = new SymbolIcon(Symbol.Next), Tag = "CardsSamplePage" });
```
Comment (NicolasChampagne) [inline src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/SamplesPage.xaml.cs:56]: can you add the menuItem in alphabetical order, to follow with the rest of the items

Comment (agneszitte-nventive) [reply src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/SamplesPage.xaml.cs:56]: Oh yeah good point, thanks @NicolasChampagne !

**Comment by MatFillion** [comment]
Definitely don't need dragged state and Swipe for now, you can remove them from your to-do on this PR :)

**Comment by agneszitte-nventive** [comment]
Updated, thanks!
Can I work on the ElevatedView in a separate PR @MatFillion please? As it will be easier to continue on the next changes.
Also for "Theming" it's for v1? did not see the theming version for the buttons for example.

File: src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/SamplesPage.xaml.cs (hunk 1 above)
Comment (agneszitte-nventive) [reply src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/SamplesPage.xaml.cs]: Done thanks !

**Comment by agneszitte-nventive** [comment]
@MatFillion I will let you take a final look at my PR before merging it

File: src/library/Uno.Material/Styles/Controls/Card.xaml [hunk 2]
```diff
@@ -1,23 +1,655 @@
...
+							<!-- Header part -->
+							<TextBlock Text="{TemplateBinding Header}"
+									   Style="{ThemeResource Headline6}" />
+							<!-- SubHeader part -->
+							<TextBlock Grid.Row="1"
+									   Text="{TemplateBinding SubHeader}"
+									   Style="{ThemeResource Body2}" />
+						</StackPanel>
+
+						<!-- Icons buttons part -->
```
[the remark this answers is not in the document]
Comment (agneszitte-nventive) [reply src/library/Uno.Material/Styles/Controls/Card.xaml:562]: Yes @MatFillion, I was unsure but I was also thinking at the re-use possibilities.
I will finish to test it with a list and will see if we have performance issues.
I will merge this PR and we can adjust after that.

**Pull request merged**